packages/ng/form-field/form-field.translate.ts

```diff
@@ -0,0 +1,19 @@
+import { InjectionToken } from '@angular/core';
+import { ILuTranslation } from '../core/translate';
+
+export const LU_FORM_FIELD_TRANSLATIONS = new InjectionToken('LuFormFieldTranslations', {
+	factory: () => luFormFieldTranslations,
+});
+
+export interface LuFormFieldTranslations {
+	counter: string;
+}
+
+export const luFormFieldTranslations: ILuTranslation<LuFormFieldTranslations> = {
+	fr: {
+		counter: 'Votre publication fait {{current}} caractères de long. {{max}} caractères maximum sont autorisés.',
+	},
+	en: {
+		counter: 'Your message is {{current}} characters long. A maximum of {{max}} is allowed.',
+	},
+};
```

Comment:
Could we add all langs? Copilot do it pretty easily

Reply:
Oh I'm not using copilot yet so yeah I didn't translate for other langs

Reply:
Small detail, but it would be even cooler with the management of plurals here.

Reply:
oof ! I think this might be out of scope but we can start to discuss it for sure.